frontend/src/views/container/container/index.vue

```diff
@@ -292,7 +292,7 @@
                     />
                     <fu-table-operations
                         fix
-                        width="180px"
+                        width="200px"
                         :ellipsis="2"
                         :buttons="buttons"
                         :label="$t('commons.table.operate')"
```

Comment:
The line change is minor but suggests an intention to widen the buttons column by 20 pixels without affecting table layout consistency across other tables in the application.

**Recommendation**: Consider updating all similar columns in this section of the code with uniform width settings. This ensures that the overall appearance remains consistent and user-friendly.

If you need further assistance or have specific parts of the code you'd like checked, feel free to share!